server.py

```diff
@@ -126,8 +126,6 @@ def assign_robot_to_client(self, ws):
         Process the data sent by interface: chosen cards,
         confirmation of selected cards, power down state, played game round.
         """
-        if robot.selection_confirmed:
```

Comment:
proč to tu mažeš?

Reply:
nevim, nepamatuju si ten kousek kodu, musela jsem ho zabit pri mergi. vratim.

Reply:
Ok :-) to je ta podmínka když se zmáčkne "k", že server dál nepřijímá zprávy.